WPMedia/arc-themes-components
# Add sign-images-in-ans-object

## Ticket

- [TMEDIA-904](https://arcpublishing.atlassian.net/browse/TMEDIA-904)

## Description

Add sign-images-in-ans-object

## Acceptance Criteria

Create a utility function that
* Inflates an ANS object with response from a cachedCall response
* Uses Promise.all to handle all requests asynchronously 

## Test Steps

1. Checkout branch - `git checkout tmedia-904`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Run tests `npm run test`
5. Check out the storybook documentation, and test file.

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
